server/methods/addAllUserToRoom.js

```diff
@@ -0,0 +1,40 @@
+Meteor.methods({
+  addAllUserToRoom: function(rid) {
+    var now, room, users;
+    var userCount = RocketChat.models.Users.find().count();
+    if(userCount > 500) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Keyword "if" must be followed by whitespace.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2777659474/issues/source?bid=3199634&fileBranchId=3362492#l5)